vsintegration/src/FSharp.Editor/Common/Extensions.fs

```diff
@@ -284,6 +285,11 @@ module Option =
         else
             None
 
+    let inline ofValueOption vo =
```

Comment:
This can be probably interchanged everywhere with ValueOption.toOption? Or is this on purpose for better readability?

Reply:
They're the same, will change